Kudu.Contracts/Jobs/FunctionTemplate.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Kudu.Contracts.Jobs
+{
+    public class FunctionTemplate
+    {
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
+
+        [JsonProperty(PropertyName = "trigger")]
+        public string Trigger { get; set; }
```

Comment:
The way the runtime handles trigger inputs, they're just another input type treated specially, so there is simply an array of Inputs (one of which is a trigger), which opens the door in the future for the possibility of multiple triggers. See: https://github.com/Azure/azure-webjobs-sdk-script/blob/master/src/WebJobs.Script/Description/FunctionMetadata.cs